packages/blockchain-wallet-v4-frontend/src/components/Form/SelectBoxBtcAddresses/selectors.ts

```diff
@@ -107,16 +107,19 @@ export const getData = (
   const toDropdown = map(x => ({ label: buildDisplay(x), value: x }))
   const toGroup = curry((label, options) => [{ label, options }])
   const toExchange = x => [{ label: `Exchange BTC Address`, value: x }]
-  const toCustodialDropdown = x => [
-    {
-      label: buildCustodialDisplay(x),
-      value: {
-        ...x,
-        type: ADDRESS_TYPES.CUSTODIAL,
-        label: 'BTC Trading Wallet'
+  const toCustodialDropdown = x => {
```

Comment:
can you use a more descriptive variable name instead of `x`?  something that indicates if its a coin, an address, etc.

Reply:
done, we had that for a while, but I agree that names are out of context :)